Library/UIView+AutoLayout.swift

```diff
@@ -45,6 +45,24 @@ public func ksr_addLayoutGuideToView() -> ((UILayoutGuide, UIView) -> (UILayoutG
   }
 }
 
+public func ksr_centerInParent(priority: UILayoutPriority = .required)
```

Comment:
How is this different from `ksr_constrainViewToCenterInParent` on line 26? 🤔

Reply:
😬 just clowning about.